File: NodeJS/simple_login_post.js
const axios = require('axios');

const loginUrl = 'URL';

const payload = {
  username: 'your_username',
  password: 'your_password',
};

axios.post(loginUrl, payload)
  .then(response => {
    console.log("Login successful!");
  })
  .catch(error => {
    console.error(`Login failed with status code: ${error.response.status}`);
  });

File: Python/simple_login_post.py
import requests

login_url = "URL"

payload = {
    'username': 'your_username',
    'password': 'your_password'
}

response = requests.post(login_url, data=payload)

if response.status_code == 200:
    print("Login successful!")
else:
    print(f"Login failed with status code {response.status_code}")
